### PR TITLE
Wip first draft

### PR DESCRIPTION
**Pull request by markedwardmurray**
@jmburges This lab turned into a small monster... but it does happen to secretly serve as a primer on how UITableViews handle their data...

This follows reading-ios-property-visibility

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,189 @@
...
+
+  * `insertWord:atIndex:inClauseAtIndex:` which takes three arguments, an `NSString` called `word`, an `NSUInteger` called `wordIndex`, and an `NSUInteger` called `clauseIndex`, and
+
+  * `replaceWordAtIndex:inClauseAtIndex:withWord:` which takes three arguments, an `NSUInteger` called `wordsIndex`, an `NSUInteger` called `clausesIndex`, and an `NSString` called `word`.
+
+#### Private Methods
+
+##### Override `init`
+
+This lesson is not about writing initializers, however the private mutable array properties must be initialized when setting up the instance of `FISSentence`. This cannot be done from the outside of the class. Paste the following code into the `FISSentence.m`; you don't to understand it for this lab but it must be included. Keep the `init` method organized at the top of the `@implementation` section as you write out the other methods:
```
Comment (jmburges) [inline README.md]: Make those two array properties public and drop this section. Feel free to allude to coolness coming up

File: objc-fissentence/FISSentence.m [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    }
+    NSString *validPunctuation = @".?!,;:—";
+    if (![validPunctuation containsString:punctuation]) {
+        return NO;
+    }
+    return YES;
+}
+
+- (BOOL)validClausesIndex:(NSUInteger)clausesIndex {
+    if (clausesIndex >= self.clauses.count) {
```
Comment (jmburges) [inline objc-fissentence/FISSentence.m:54]: Condense this into one if

**Comment by jmburges** [comment]
Few changes and :shipit:

File: objc-fissentence/FISSentence.m (hunk 2 above)
Comment (markedwardmurray) [inline objc-fissentence/FISSentence.m:54]: are you sure? `containsString:` throws an exception when the argument is `nil`

Comment (jmburges) [inline objc-fissentence/FISSentence.m:54]: Good point. disregard

**Pull request merged**